### ISSUE TITLE
Location property synched to remote calendar, but now backwards

### ISSUE THREAD

**Issue by rgemulla** [dengste/org-caldav#123]
When an org-entry has a LOCATION property, org-caldav syncs the the entry correctly to the remote calendar (both creation, changes, and deletion of the location). 

The other way around does not work, however: changes in the calendar are not picked up at all by org-caldav.

**Comment by grauschnabel** [comment]
Yes, that's how it's written in the readme.

**Comment by rgemulla** [comment]
I understood the README as follows: at the time the README was written, org-icalendar seemed to have ignored the location property. Now it extracts it correctly (it's in the ics file and thus will be synched). Just not backwards.

**Comment by grauschnabel** [comment]
Yes, someone has to implement it into org-caldav. Should not  be to hard, but someone has to do it.

**Comment by rgemulla** [comment]
I'll give it a shot.

**Comment by rgemulla** [comment]
I've attached a modification that syncs locations for any value of org-caldav-sync-changes-to-org (i.e., title-and-timestamp, title-only, timestamp-only, all). 

Only minor modifications were needed; e.g., adding a an additional location element to eventdata.

Feel free to incorporate this change into org-caldav.

[rgmacs-org-caldav-location.zip](https://github.com/dengste/org-caldav/files/1232282/rgmacs-org-caldav-location.zip)

**Comment by grauschnabel** [comment]
thanks for that, I will look at this later, but maybe you could do it with a pull request next time?  That would be much easier to put it into it.

**Comment by rgemulla** [comment]
Sure! 

This time I didn't edit org-caldav directly, but just overwrote some functions in a separate file that I load on startup. This was easiest for me, but I agree that this increases the burden of putting it in. Next time I'll use a pull request.

**Comment by rgemulla** [comment]
Any chance to get this integrated into org-caldav? I created pull request #124 to make things easier.

**Comment by joranvar** [comment]
Sorry for the reference-spam.  I tried to rewrite @rgemulla's commit to remove the whitespace changes.  Didn't notice that the many pushes and rebases would lead to this.

**Comment by Titan-C** [comment]
Is there any progress on this? Is it working?

**Comment by grauschnabel** [comment]
I think, not at the moment. How important is that for you?

-- 
Mag. Martin Kaffanke
+43 650 4514224
http://www.kaffanke.at

-----Original Message-----

**Comment by rgemulla** [comment]
Well, it's implemented in PR #124 and working on my machine without any problems since a couple of month. Unfortunately, org-caldav does not seem to be maintained anymore to that you may have to integrate that PR yourself if you want this functionality.

**Comment by Titan-C** [comment]
I just moved to a new city and having location appointments is becoming
relevant.
I was fine org-caldav not syncing location as it says on the readme, I
would painfully load the events through the web interface when
needed. My issues arises because caldav syncs from server to local
file and skips the location, but the next sync it pushes/updates to server
the events without location. Then I no more have the location available in
the server and my other caldav client on the phone has no information of
the location of the event.

**Comment by rgemulla** [comment]
As I mentioned, if you apply PR #124, location synchronization works both ways.

Another option is to require the content of the [rgmacs-org-caldav-location.zip](https://github.com/dengste/org-caldav/files/1232282/rgmacs-org-caldav-location.zip) after loading org-caldav.

**Comment by Titan-C** [comment]
I applied pr #124 and is working. Thank you @rgemulla

**Comment by bepolymathe** [comment]
For me it doesn't work for synchronization with Nextcloud. Can it come from the interpretation by the Nextcloud server?
cal --» org (OK)
org --» cal (Not work)

The format is : 
```
:PROPERTIES:
  :ID:       Nextcloud-TB9J700F1J683DMH2QL8R
  :LOCATION: Adress
  :END:
  <2019-10-06 dim. 14:00-15:00>

```

**Comment by bepolymathe** [comment]
Could you confirm the issue ?

**Comment by Thaodan** [comment]
I wonder if there should be separate setting to sync location only.
